### PR TITLE
Create bricekams.md

### PR DESCRIPTION
**Pull request by bricekk**
new contributor osscameroon

### CODE REVIEW

File: content/post/contributors/bricekams.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: "Brice Kams"
+date: 2022-02-19T13:01:54+01:00
+authors: ["Brice Kams"]
+---
+
+Hi, am Brice Kams, i'm learning code to be a great softwware developper.
```
Comment (RMPR) [inline content/post/contributors/bricekams.md:7]:
```suggestion
Hi, I am Brice Kams, I'm learning code to be a great software developer.
```

Comment (bricekk) [reply content/post/contributors/bricekams.md:7]: Oh tanks

Comment (RMPR) [reply content/post/contributors/bricekams.md:7]: https://github.com/osscameroon/osscameroon-blog/pull/90#discussion_r867314348

Comment (bricekk) [reply content/post/contributors/bricekams.md]: thanks again.

**Review by RMPR — APPROVED** [review]

**Pull request merged**